src/mono/wasm/wasm.proj

```diff
@@ -108,17 +108,50 @@
     <Error Text="Failed to parse emcc version, and hash from the full version string: '$(_EmccVersionRaw)'"
            Condition="'$(_EmccVersion)' == '' or '$(_EmccVersionHash)' == ''" />
 
+    <!-- keep in sync with src\mono\wasm\build\WasmApp.Native.targets -->
+    <ItemGroup>
+      <EmccExportedRuntimeMethod Include="FS" />
+      <EmccExportedRuntimeMethod Include="print" />
```

Comment:
Also, add all these default ones to `emcc-props.json`.

1. Add the items here: https://github.com/dotnet/runtime/blob/1281a4a3393e9293aa4b379d619a692d6dcb62a4/src/mono/wasm/wasm.proj#L159-L174

2. Add the items for reading:
https://github.com/dotnet/runtime/blob/28c7eb85636d8c97d44b0a619d59eb98ba0b98e3/src/mono/wasm/build/WasmApp.Native.targets#L639-L645

3. Then you can use the new items